Factories/RollbarHandlerFactory.php

```diff
@@ -30,30 +35,26 @@ public function __construct(ContainerInterface $container)
 
         if (!empty($config['person_fn']) && is_callable($config['person_fn'])) {
             $config['person'] = null;
-        } else {
-            
-            if (empty($config['person'])) {
-                
-                $config['person_fn'] = function() use ($container) {
-                    
-                    try {
-                        $token = $container->get('security.token_storage')->getToken();
-                        
-                        if ($token) {
-                            $user = $token->getUser();
-                            $serializer = $container->get('serializer');
-                            $person = \json_decode($serializer->serialize($user, 'json'), true);
-                            return $person;
-                        }
-                    } catch (\Exception $exception) {
-                        // Ignore
+        } elseif (empty($config['person'])) {
+            $config['person_fn'] = function () use ($container) {
+
+                try {
+                    $token = $container->get('security.token_storage')->getToken();
+
+                    if ($token) {
+                        $user = $token->getUser();
+                        $serializer = $container->get('serializer');
+
+                        return \json_decode($serializer->serialize($user, 'json'), true);
                     }
-                };
-                
-            }
-            
+                } catch (\Exception $exception) {
+                    // Ignore
+                }
+            };
         }
 
+        $this->minimumLevel = $config['minimum_level'] ?: LogLevel::ERROR;
```

Comment:
Same here, I recommend using `monolog_minimum_level` for the config option name as suggested in https://github.com/rollbar/rollbar-php/pull/440#pullrequestreview-215372893

Also, I think `isset($config['minimum_level'])` is needed here as I get PHP notices if `minimum_level` is not set in `rollbar.yaml`:

```php
$this->minimumLevel = isset($config['minimum_level']) ? $config['minimum_level'] : LogLevel::ERROR;
```

Reply:
@ArturMoczulski okay, i change it.